### DISCUSSION TITLE
course/08

### DISCUSSION THREAD

**Discussion by giscus** [mitre/inspec-advanced-developer#3] [Ideas]
# course/08

The MITRE Security Automation Framework Team's Advanced InSpec profile development course

https://deploy-preview-2--mitre-inspec-advanced-developer.netlify.app/course/08.html

**Comment by aaronlippold** [comment]
Use the `inspec init resource ...` to create the shell for the demo `git` resource, and also for the plural other examples.

**Reply by aaronlippold** [reply]
You may need to update the 'input command' and 'output result' given the above.

**Comment by karikarshivani** [comment]
Consider changing the name of the profile from `git` to `my_git_profile` so that it's easily noticeable. Maybe `git_test` zip folder can also be renamed to `target_directory` or something descriptive. That way, students can navigate easily between `git.rb` resource, the profile, and the path of evaluation.
